data/102/048/635/102048635.geojson

```diff
@@ -10,6 +10,12 @@
     "geom:latitude":-35.827657,
     "geom:longitude":146.262864,
     "iso:country":"AU",
+    "label:eng_x_preferred_longname":[
+        "Corowa Shire Council"
+    ],
+    "label:eng_x_preferred_placetype":[
+        "council"
```

Comment:
Why is this `council` and not `shire council` (repeats)?

Reply:
Interesting case - The "Corowa Shire" portion is sourced from the `wof:name`, the "Council" comes from the appended placetype in the `qs:a2` property. It looks like both versions of the name, "Corowa Shire" and "Corowa Shire Council" are legitimate, however..

https://en.wikipedia.org/wiki/Corowa_Shire

It looks like this feature recently merged with neighboring shires. I'll leave this value as is and take care of the superseding/deprecating in a follow-up PR.